.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "extensionHost",
+      "request": "launch",
+      "name": "Debug LSP Extension",
+      "runtimeExecutable": "${execPath}",
+      "args": [
+        "--extensionDevelopmentPath=${workspaceRoot}/client/vscode",
+        "--disable-extensions",
+        "${workspaceRoot}/htmx.test.html"

```

Comment:
i don't get this argument

Reply:
Has the debugging VSCode window open the file `htmx.test.html` by default when launched